village/src/components/Smurfs.js

```diff
@@ -1,32 +1,36 @@
-import React, { Component } from 'react';
+import React, { Component } from "react";
 
-import Smurf from './Smurf';
+import Smurf from "./Smurf";
 
 class Smurfs extends Component {
   render() {
     return (
       <div className="Smurfs">
         <h1>Smurf Village</h1>
         <ul>
-          {this.props.smurfs.map(smurf => {
-            return (
+          {this.props.smurfs.map(smurf => (
+            // ? I put the key here in order to get rid of error
+            <div key={smurf.id}>
```

Comment:
If you return the component without a div wrapped around it, the key on line 19 should work. If you have a div wrapping the component, the div is the wrapper that needs a key.